Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -229,7 +229,14 @@ static void SetSamplerState(u32 index, bool custom_tex)
 
   // Custom textures may have a greater number of mips
   if (custom_tex)
+  {
     state.max_lod = 255;
+  }
+  else if(has_manual_mips && SamplerCommon::AreBpTexMode0MipmapsEnabled(tm0))
+  {
+    //note: this is enabled even at 1x IR because we also want to disable Anisotropic for those textures
+    state.lod_bias_scale = g_ActiveConfig.iEFBScale;
```

Comment:
What about just enforcing state.anisotropic_filtering = 0 here?

Reply:
I saw a lot of places where the rest of the code changes the `anisotropic` so to be 100% sure I just had `lod_bias_scale` disable `anisotropic` in the Render.
Not only that, but doing stuff like never entering the `if(maxAnisotropy != 0)` that's just below (tried that first) seems to also make you fallback to point filtering... so the answer is "because this worked" :P 
I'll give it some more thought, limiting the changes to each backend is better.